### PR TITLE
upgrade to ff/group 0.13.0

### PR DESCRIPTION
**Pull request by zhenfeizhang**

### CODE REVIEW

**Review by naure — CHANGES_REQUESTED** [review]
LGTM besides couple of comments.

**Comment by naure** [comment]
I resolved the conflict with recent changes in `main`. Please let me know whether that works for you.

**Comment by noel2004** [comment]
Now I have merged the main branch (with septidon refactoring) in it and the code seems work now

For the special dependency of halo2 lib (`halo2-ecc-snark-verifier-0220`), I think it is still not able to be merged into main branch. Just kept it here until our halo2 lib dependency become stable

File: rust-toolchain [hunk 1]
```diff
@@ -1 +1 @@
-nightly-2022-08-23
\ No newline at end of file
+nightly
```
[the remark this answers is not in the document]
Comment (noel2004) [reply rust-toolchain]: I think this should work.

**Comment by zhenfeizhang** [comment]
replaced with #29

**Pull request closed without merging**